src/desktop/space/element.rs

```diff
@@ -163,6 +112,9 @@ where
         damage: &[Rectangle<i32, Logical>],
         log: &slog::Logger,
     ) -> Result<(), R::Error>;
+    fn z_index(&self) -> u8; //{
```

Comment:
RenderElements have a default, look above and for the implementation of SpaceElement for them, the default to RenderZindex::Overlay.
That is the default for SpaceElement overall, I did have a default for it but comment it out to find where I need implement it, but yes it will be better without one